test/e2e/case2_config_deployment_test.go

```diff
@@ -143,7 +143,7 @@ var _ = Describe("Test config-policy-controller deployment", func() {
 
 			By(logPrefix +
 				"removing the config-policy-controller deployment when the ManagedClusterAddOn CR is removed")
-			Kubectl("delete", "-n", cluster.clusterName, "-f", case2ManagedClusterAddOnCR, "--timeout=30s")
+			Kubectl("delete", "-n", cluster.clusterName, "-f", case2ManagedClusterAddOnCR, "--timeout=90s")
 			deploy = GetWithTimeout(
 				cluster.clusterClient, gvrDeployment, case2DeploymentName, addonNamespace, false, 180,
```

Comment:
See previous comment... I updated most of the 180s in my local testing.  I have still hit some other timeout locally so will try your changes with mine to see if I can find any other suggestions.